### ISSUE TITLE
404 Client Error: Not Found for url: https://pypi.org//pypi

### ISSUE THREAD

**Issue by carlthome** [conda/conda-build#1357]
Cannot generate PyPI recipes with conda skeleton due to a 404.

# conda skeleton pypi numpy
```
INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): pypi.io
INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): pypi.org
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda-build/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/home/carl/anaconda3/bin/conda-skeleton", line 6, in <module>
    sys.exit(conda_build.main_skeleton.main())
  File "/home/carl/anaconda3/lib/python3.5/site-packages/conda_build/main_skeleton.py", line 319, in main
    args_func(args, p)
  File "/home/carl/anaconda3/lib/python3.5/site-packages/conda_build/main_build.py", line 409, in args_func
    args.func(args, p)
  File "/home/carl/anaconda3/lib/python3.5/site-packages/conda_build/main_skeleton.py", line 334, in execute
    pypi.main(args, parser)
  File "/home/carl/anaconda3/lib/python3.5/site-packages/conda_build/pypi.py", line 268, in main
    all_packages = [match["name"] for match in client.search({"name": args.packages}, "or")]
  File "/home/carl/anaconda3/lib/python3.5/xmlrpc/client.py", line 1092, in __call__
    return self.__send(self.__name, args)
  File "/home/carl/anaconda3/lib/python3.5/xmlrpc/client.py", line 1432, in __request
    verbose=self.__verbose
  File "/home/carl/anaconda3/lib/python3.5/site-packages/conda_build/pypi.py", line 207, in request
    resp.raise_for_status()
  File "/home/carl/anaconda3/lib/python3.5/site-packages/requests/models.py", line 862, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org//pypi
```

# conda info
```sh
Current conda install:

             platform : linux-64
        conda version : 4.1.11
    conda-env version : 2.5.2
  conda-build version : 1.21.8
       python version : 3.5.2.final.0
     requests version : 2.11.1
     root environment : /home/carl/anaconda3  (writable)
  default environment : /home/carl/anaconda3
     envs directories : /home/carl/anaconda3/envs
        package cache : /home/carl/anaconda3/pkgs
         channel URLs : https://conda.anaconda.org/t/<TOKEN>/conda-forge/linux-64/
                        https://conda.anaconda.org/t/<TOKEN>/conda-forge/noarch/
                        https://conda.anaconda.org/t/<TOKEN>/pypi/linux-64/
                        https://conda.anaconda.org/t/<TOKEN>/pypi/noarch/
                        https://conda.anaconda.org/t/<TOKEN>/carlthome/linux-64/
                        https://conda.anaconda.org/t/<TOKEN>/carlthome/noarch/
                        https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /home/carl/.condarc
         offline mode : False
    is foreign system : False
```

**Comment by msarahan** [comment]
There is nothing in the current conda-build codebase that has that URL.  Please update conda-build and let us know if the problem persists.

**Comment by carlthome** [comment]
```
# All requested packages already installed.
# packages in environment at /home/carl/anaconda3:
#
conda-build               1.21.8                   py35_0 
```

**Comment by msarahan** [comment]
It seems like you might be running

```
conda install conda-build
```

when you need to be running

```
conda update conda-build
```

Unless you are on a corporate network, I don't know of a reason why you'd be getting such an outdated version.  1.21.8 was released on July 31, and the current version is 2.0.1.

**Comment by carlthome** [comment]
Very weird indeed. I've reinstalled Anaconda 3. Thanks for the help!
